test/crd.go

```diff
@@ -35,6 +36,8 @@ type ResourceNames struct {
 	TrafficTarget string
 	URL           *url.URL
 	Image         string
+	// To support multi container related testcases
+	Containers []corev1.Container
```

Comment:
This is not exactly a resource name, is it? :thinking:

Reply:
Ya i agree your point modified as per the suggestion to pass via `ServiceOptions`
Thank you